packages/js/product-editor/src/components/iframe-editor/iframe-editor.scss

```diff
@@ -0,0 +1,19 @@
+.woocommerce-iframe-editor {
+    align-items: center;
+    background-color: #2f2f2f;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+    overflow: hidden;
+    position: fixed;
```

Comment:
This is currently fixed, but we may want to slot fill this somewhere else into the page to position absolutely and prevent overlap with the existing editor.

More investigation needed here to determine the right solution.